stars/hip_gaia3/name.fab

```diff
@@ -1,4943 +1,5108 @@
- 69427|κ_Vir
- 69974|λ_Vir
- 71957|μ_Vir
- 57380|ν_Vir
- 57328|ξ_Vir
- 58948|ο_Vir
- 58590|π_Vir
- 61960|ρ_Vir
- 64852|σ_Vir
- 68520|τ_Vir
- 70012|υ_Vir
- 70755|φ_Vir
- 61740|χ_Vir
- 62985|ψ_Vir
- 56779|ω_Vir
- 58510|b_Vir
- 60172|c_Vir
- 61968|d1_Vir
- 62267|d2_Vir
- 64792|e_Vir
- 61558|f_Vir
- 64122|g_Vir
- 66098|h_Vir
- 65581|i_Vir
- 63414|k_Vir
- 66006|l_Vir
- 66803|m_Vir
- 66200|o_Vir
- 67929|p_Vir
- 61318|q_Vir
- 66247|y_Vir
- 57562|A1_Vir
- 58110|A2_Vir
- 73193|M_Vir
- 56779|1_Vir
- 57328|2_Vir
- 57380|3_Vir
- 57562|4_Vir
- 57757|5_Vir
- 58110|6_Vir
- 58510|7_Vir
- 58590|8_Vir
- 58948|9_Vir
- 59285|10_Vir
- 59309|11_Vir
- 59608|12_Vir
- 60030|13_Vir
- 60088|14_Vir
- 60129|15_Vir
- 60172|16_Vir
- 60353|17_Vir
- 61246|20_Vir
- 61318|21_Vir
- 61558|25_Vir
- 61740|26_Vir
- 61937|27_Vir
- 61969|28_Vir
- 61941|29_Vir
- 61960|30_Vir
- 61968|31_Vir
- 62267|32_Vir
- 62325|33_Vir
- 62394|34_Vir
- 62443|35_Vir
- 62541|36_Vir
- 62757|37_Vir
- 62875|38_Vir
- 62985|40_Vir
- 62933|41_Vir
- 63090|43_Vir
- 63414|44_Vir
- 63494|46_Vir
- 63608|47_Vir
- 63750|48_Vir
- 64078|49_Vir
- 64224|50_Vir
- 64238|51_Vir
- 64407|53_Vir
- 64520|54_Vir
- 64577|55_Vir
- 64625|56_Vir
- 64725|57_Vir
- 64834|58_Vir
- 64792|59_Vir
- 64852|60_Vir
- 64924|61_Vir
- 65074|62_Vir
- 65301|63_Vir
- 65241|64_Vir
- 65323|65_Vir
- 65420|66_Vir
- 65474|67_Vir
- 65581|68_Vir
- 65639|69_Vir
- 65721|70_Vir
- 65790|71_Vir
- 65892|72_Vir
- 66015|73_Vir
- 66006|74_Vir
- 66091|75_Vir
- 66098|76_Vir
- 66131|77_Vir
- 66200|78_Vir
- 66249|79_Vir
- 66320|80_Vir
- 66465|81_Vir
- 66803|82_Vir
- 67057|83_Vir
- 66936|84_Vir
- 67139|85_Vir
- 67172|86_Vir
- 67288|87_Vir
- 67355|88_Vir
- 67494|89_Vir
- 67929|90_Vir
- 68092|92_Vir
- 68520|93_Vir
- 68888|94_Vir
- 68940|95_Vir
- 69127|96_Vir
- 69403|97_Vir
- 69427|98_Vir
- 69701|99_Vir
- 69974|100_Vir
- 69829|101_Vir
- 70012|102_Vir
- 70220|103_Vir
- 70680|104_Vir
- 70755|105_Vir
- 70794|106_Vir
- 71957|107_Vir
- 72154|108_Vir
- 72220|109_Vir
- 73620|110_Vir
- 69929|236_Vir
- 44382|α_Vol
- 41312|β_Vol
- 34473|γ1_Vol
- 34481|γ2_Vol
- 35228|δ_Vol
- 39794|ε_Vol
- 37504|ζ_Vol
- 41003|η_Vol
- 42425|θ_Vol
- 32912|ι_Vol
- 40817|κ1_Vol
- 40834|κ2_Vol
- 95771|α_Vul
- 94703|1_Vul
- 94827|2_Vul
- 95260|3_Vul
- 95498|4_Vul
- 95560|5_Vul
- 95771|6_Vul
- 95818|7_Vul
- 95785|8_Vul
- 96275|9_Vul
- 97077|10_Vul
- 97679|12_Vul
- 97886|13_Vul
- 98375|14_Vul
- 98543|15_Vul
- 98636|16_Vul
- 99080|17_Vul
- 99404|18_Vul
- 99518|19_Vul
- 99531|20_Vul
- 99738|21_Vul
- 99853|22_Vul
- 99874|23_Vul
- 99951|24_Vul
-100435|25_Vul
-101641|26_Vul
-101716|27_Vul
-101868|28_Vul
-101867|29_Vul
-102388|30_Vul
-103004|31_Vul
-103200|32_Vul
-103511|33_Vul
-105411|34_Vul
-105966|35_Vul
-5350358584482202880|η Car
+# HIP/Gaia3        |Bayer/Flamsteed designations
+# IAU constellation Andromeda (And)
+                677|α_And
+                841|22_And
+               5447|β_And
+               9640|γ1_And
+ 346231302441905920|γ2_And
+ 346231302441905920|γ_And_B
+ 346231302441905920|γ_And_C
```

Comment:
Is this really the same star, or can two stars share the same Gaia number? If component C is spatially unseparable from B, maybe a single BC entry is better.

Reply:
I think just a B will be enough here